Sources/Helpers/APIVersion.swift

```diff
@@ -48,12 +48,14 @@ extension ZMAPIVersion {
 
 }
 
-// MARK: - Comparable
+// MARK: - Find the highest common version
 
-extension ZMAPIVersion: Comparable {
+extension APIVersion {
 
-    public static func < (lhs: Self, rhs: Self) -> Bool {
-        return lhs.rawValue < rhs.rawValue
+    public static func highestCommonVersion(with versions: [Int32]) -> Self? {
+        versions.sorted(by: >)
+            .compactMap { APIVersion(rawValue: Int32($0)) }
+            .first
```

Comment:
An alternate way to express this, maybe slightly more readable?

```suggestion
        versions
            .compactMap { APIVersion(rawValue: Int32($0)) }
            .max()
```